packages/react-data-grid/src/Canvas.js

```diff
@@ -246,28 +250,23 @@ class Canvas extends React.PureComponent {
     return this.props.rowHeight * rowIdx;
   }
 
-  getSelectedRowHeight = (rowIdx) => {
```

Comment:
I believe this logic was added to support variable height rows. This adds additional complexity and it is currently not used by `Copy` and `Drag` masks

Reply:
by saying variable height, do you mean `row1` is 50 and `row2` is 35 ?

Reply:
That is correct. This is a great feature to have but we need to handle all the masks (Copy and Drag). This currently does not work with them and has some edge cases so I removed it

Reply:
I agree that it adds complexity. When starting the work on the Interaction Mask, I did it under the assumption that RDG only supports fixed row height. However, this is not exactly true as it is possible to have dynamic row height using custom row renderers. I have seen a lot of cases out in the wild where this is happening. Removing this, is going to cause a lot of strange display issues for those cases

Supporting dynamic row height is not trivial, and this current implementation was really just a stop gap to support these non documented edge cases. I would prefer to leave this here and think of a better solution that to completely remove it